### ISSUE TITLE
Honda Bosch with alternate brake message 0x1BE does not validate checksum/counter

### ISSUE THREAD

**Issue by gregjhogan** [commaai/panda#536]
The brake pedal press bit used by most honda bosch vehicles was never added to the list of RX messages for checksum/counter validation.
https://github.com/commaai/panda/blob/01bf7402401dc5e38b3172628dfb1e4491e52b84/board/safety/safety_honda.h#L27-L41

It can't simply be added as a new, separate message to validate, though, because some honda bosch vehicles (civic hatch) do not have a message broadcast on address 0x1BE.

I believe the following is needed:
* honda bosch vehicles without alt brake message validate 0x17C for gas and brake (no change)
* honda bosch vehicles with alt brake message validate 0x17C for gas and 0x1BE for brake

There is a test ready for this once it is implemented:
https://github.com/commaai/panda/blob/01bf7402401dc5e38b3172628dfb1e4491e52b84/tests/safety/test_honda.py#L277-L285

**Comment by sshane** [comment]
Yes, the PR to catch this is not yet in though: https://github.com/commaai/panda/pull/1732
